hack/release.toml

```diff
@@ -111,6 +111,13 @@ In this mode the config change is applied for a period of time and then reverted
 This new mode can be used only with the parts of the config that can be changed without a reboot and can help to check that
 the new configuration doesn't break the node.
 Can be especially useful to check network interfaces changes that may lead to the loss of connectivity to the node.
+"""
+
+    [notes.network-device-selector]
+        title = "Network Device Selector"
+        description="""\
+Add `deviceSelector` parameter to the `.machine.network.interfaces`, which can be used to select an interface using `kernelDriver`,
```

Comment:
```suggestion
Talos machine configuration supports specifying network interfaces by selectors instead of interface name. <there should be a link to the docs describing the feature, how to use it, examples, etc.>
```